tests/unit/linalg/test_matrix_dense.cpp

```diff
@@ -15,6 +15,25 @@
 
 using namespace mfem;
 
+TEST_CASE("DenseMatrix init-list construction", "[DenseMatrix]")
+{
+   double ContigData[6] = {6.0, 5.0,
+                           4.0, 3.0,
+                           2.0, 1.0
+                          };
+   DenseMatrix Contiguous(ContigData, 2, 3);
+
+   DenseMatrix Nested({{6.0, 4.0, 2.0}, {5.0, 3.0, 1.0}});
```

Comment:
```suggestion
   DenseMatrix Nested(
   {
      {6.0, 4.0, 2.0},
      {5.0, 3.0, 1.0}
   });
```
Might be easier to see the matrix this way… small nitpick.